charterino/charterino/VisualizeDataPage.xaml.cs

```diff
@@ -20,6 +21,8 @@ namespace charterino
     /// </summary>
     public partial class VisualizeDataPage : Page
     {
+        List<Product> data = new List<Product>();
```

Comment:
Co do za nieużywana lista?

Reply:
To twoja lista.

Reply:
nie jest moja. Jest twoja

Reply:
Zrobiłem konstruktor do twojego produktu :)

Reply:
> Zrobiłem konstruktor do twojego produktu :)

spoko spoko, suuper!